src/breakpoints/index.ts

```diff
@@ -52,13 +67,76 @@ class BreakpointsHeader extends Widget {
 }
 
 export namespace Breakpoints {
+  export interface IBreakpoint extends DebugProtocol.Breakpoint {}
```

Comment:
Maybe this could be a type alias for now if we don't plan to extend the `DebugProtocol.Breakpoint` interface?

```typescript
export type IBreakpoint = DebugProtocol.Breakpoint;
```

Reply:
Good question. Hmm... It depends on how actually debugger will be work. If we don't wanna implement own UI functionality for breakpoints it's ok.  I will think about this.